environments/hca-prod/terraform/buckets.tf

```diff
@@ -0,0 +1,245 @@
+}
+
+resource google_storage_bucket_iam_member hca_argo_staging_bucket_iam {
+  provider = google-beta.target
+  bucket =  google_storage_bucket.staging_storage.name
+  # When the storage.admin role is applied to an individual bucket,
+  # the control applies only to the specified bucket and objects within
+  # the bucket: https://cloud.google.com/storage/docs/access-control/iam-roles
+  role = "roles/storage.admin"
+  member = "serviceAccount:${module.hca_argo_runner_account.email}"
+}
+
+resource google_storage_bucket_iam_member staging_account_iam_reader {
+  provider = google-beta.target
+  bucket = google_storage_bucket.staging_storage.name
+  # Object viewer gives both 'list' and 'get' permissions to all objects in the bucket.
+  role = "roles/storage.objectViewer"
+  member = "serviceAccount:${local.prod_repo_email}"
+}
+
+# Bucket for long term Argo logs storage, currently want no "delete after N days" rule.
+resource google_storage_bucket hca_argo_archive {
+  provider = google-beta.target
+  name = "${local.prod_project_name}-argo-archive"
+  location = "US"
+}
+
+resource google_storage_bucket_iam_member hca_argo_logs_bucket_iam {
+  provider = google-beta.target
+  bucket =  google_storage_bucket.hca_argo_archive.name
+  # When the storage.admin role is applied to an individual bucket,
+  # the control applies only to the specified bucket and objects within
+  # the bucket: https://cloud.google.com/storage/docs/access-control/iam-roles
+  role = "roles/storage.admin"
+  member = "serviceAccount:${module.hca_argo_runner_account.email}"
+}
+# Service accounts that use these buckets
+# sa w/permissions to use dataflow & bigquery
+module hca_dataflow_account {
+  source = "/templates/google-sa"
+  providers = {
+    google.target = google-beta.target,
+    vault.target = vault.target
+  }
+
+  account_id = "hca-dataflow-runner"
+  display_name = "Service account to run HCA dataflow jobs"
+  vault_path = "${local.prod_vault_prefix}/service-accounts/hca-dataflow-runner"
+  roles = ["dataflow.worker"]
+}
+
+module hca_argo_runner_account {
+  source = "/templates/google-sa"
+  providers = {
+    google.target = google-beta.target,
+    vault.target = vault.target
+  }
+
+  account_id = "hca-argo-runner"
+  display_name = "Service account to run HCA's Argo workflow."
+  vault_path = "${local.prod_vault_prefix}/service-accounts/hca-argo-runner"
+  roles = ["dataflow.developer", "compute.viewer", "bigquery.jobUser", "bigquery.dataOwner"]
+}
+
+data google_project current_project {
+  provider = google-beta.target
+}
+
+//resource google_service_account_iam_binding hca_workload_identity_binding {
+//  provider = google-beta.target
+//
+//  service_account_id = module.hca_argo_runner_account.id
+//  role = "roles/iam.workloadIdentityUser"
+//  members = ["serviceAccount:${data.google_project.current_project.id}.svc.id.goog[hca/argo-runner]"]
+//  depends_on = [module.hca_argo_runner_account]
+//}
```

Comment:
You intended to leave this commented out, right?

Reply:
yep, made a ticket to fix it later